meinberlin/apps/plans/models.py

```diff
@@ -115,6 +116,7 @@ class Plan(ProjectContactDetailMixin, UserGeneratedContentModel):
         blank=True,
         help_prefix=_("Visualize your plan with an image underneath the description."),
     )
+    description_image_alt_text = ImageAltTextField(image_name=_("Header image"))
     description_image_copyright = models.CharField(
         verbose_name=_("Header image copyright"),
         blank=True,
```

Comment:
again we are making a new field instead of importing from a4?

Reply:
Hmmja, we should have used the one from a4. Should still do that and migrate the data, then. But we do use it for the tile image, so maybe there is some reason? I don't know.